run-tests.php

```diff
@@ -880,20 +875,19 @@ function write_information(): void
     // load list of enabled and loadable extensions
     save_text($info_file, <<<'PHP'
         <?php
-        echo str_replace("Zend OPcache", "opcache", implode(",", get_loaded_extensions()));
-        $ext_dir = ini_get("extension_dir");
+        $exts = get_loaded_extensions();
+        $ext_dir = ini_get('extension_dir');
         foreach (scandir($ext_dir) as $file) {
-            if (!preg_match('/^(?:php_)?([_a-zA-Z0-9]+)\.(?:so|dll)$/', $file, $matches)) {
-                continue;
-            }
-            $ext = $matches[1];
-            if (!extension_loaded($ext) && @dl($file)) {
-                echo ",", $ext;
+            if (preg_match('/^(?:php_)?([_a-zA-Z0-9]+)\.(?:so|dll)$/', $file, $matches)) {
+                if (@dl($matches[1])) {
+                    $exts[] = $matches[1];
+                }
             }
         }
-        ?>
-    PHP);
-    $exts_to_test = explode(',', shell_exec("$php $pass_options $info_params $no_file_cache \"$info_file\""));
+        echo implode(',', $exts);
+        PHP);
```

Comment:
We should probably "detend" this line:
```suggestion
    PHP);
```

Reply:
why? with the new change, the code has (as I would expect) no indentation

Reply:
Shouldn't this line match https://github.com/php/php-src/pull/8363/files#diff-d04552a469f6d782895191695156cf282c5577963770c9acf2479cbacc03672cR876?

Reply:
I am sorry, what do you mean?

Reply:
The closing `PHP` tag should match the indentation of the opening `'PHP'`.

Reply:
Why? The indentation is determined solely by the ending `PHP` tag.

You meant to remove 1 indentation more for whole nowdoc?

Reply:
Oh, sorry, you're right. The indentation is fine as is.